roles/manage-aws-infra/defaults/main.yml

```diff
@@ -132,13 +143,15 @@ etcd_root_volume_size: "{{ cloud_infrastructure.etcdnodes.root_volume_size  }}"
 infra_node_root_volume_size: "{{ cloud_infrastructure.infranodes.root_volume_size }}"
 app_node_root_volume_size: "{{ cloud_infrastructure.appnodes.root_volume_size }}"
 cns_node_root_volume_size: "{{ cloud_infrastructure.cnsnodes.root_volume_size }}"
+bastion_node_root_volume_size: "{{ cloud_infrastructure.bastion.root_volume_size }}"
 
 ## Feed root volume device sizes for each instance from `cloud_infrastructure` list on the inventory
 master_docker_volume_size: "{{ cloud_infrastructure.masters.docker_volume_size }}"
 etcd_docker_volume_size: "{{ cloud_infrastructure.etcdnodes.docker_volume_size  }}"
 infra_node_docker_volume_size: "{{ cloud_infrastructure.infranodes.docker_volume_size }}"
 app_node_docker_volume_size: "{{ cloud_infrastructure.appnodes.docker_volume_size }}"
 cns_node_docker_volume_size: "{{ cloud_infrastructure.cnsnodes.docker_volume_size }}"
+bastion_node_docker_volume_size: "{{ cloud_infrastructure.bastion.docker_volume_size }}"
```

Comment:
The inventory skeleton is not adding any volume for Docker in the Bastion Nodes

Reply:
Fixed